src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -23,6 +23,23 @@ import { useActiveWeb3React } from 'hooks/web3'
 import InvestOption from './InvestOption'
 import { ClaimCommonTypes, ClaimWithInvestmentData, EnhancedUserClaimData } from '../types'
 
+const STEPS_DATA = [
+  {
+    id: 0,
+    title: 'Start',
+  },
+  {
+    id: 1,
```

Comment:
why do you need an id if the index/order is deterministic? it's an array

Reply:
My thinking was that this would be a more fool proof approach. Order of an `Array` in the context of this small Array of objects `const` indeed is deterministic. 

In context of a general purpose `Stepper` component, my understanding is that in scenarios where you might have missing (object) values in the Array or externally imported data, one could argue setting explicit step numbers might be a defensive way of setting it up, given the order/index number might be affected in those scenarios. But correct me if I'm wrong.

Happy to do it another way, but just to highlight.

Reply:
Feel free to review post merge. For now I reverted logic back to use `index` for key.

Reply:
Then make id optional as a prop and use the index as a fallback

Reply:
Sure Michel, makes sense if you plan to feed this with dynamic data which doesn't have order guarantees. 
My thinking was, it's just using a static deterministic array, and in practical terms it will always be the case. I don't see the stepper consuming a dynamic list from an untrusted REST API  :P 

Anyways, it's all good, i consider the comment a ubernit :) 
Fine to leave it, fine to remove it, fine to make it optional

Reply:
I see, could also be an option. A bit messy potentially in the end where `index` and `id` are thrown together and mixed up? I think I leave it as per your first suggestion and use only `index` for now.

Reply:
@anxolin Yeah, probably I'm over thinking it at this moment :)